Team9Project/src/com/indragie/cmput301as1/ExpenseClaimListActivity.java

```diff
@@ -63,6 +64,16 @@
 	private ListModel<ExpenseClaim> listModel;
 	
 	/**
+	 * List Model of filtered expense claim.
+	 */
+	private ListModel<ExpenseClaim> filteredListModel;
+	
+	/**
+	 * List of tags to filter expense claims.
+	 */
+	private ArrayList<Tag> filteredTagsList;
```

Comment:
I don't think so, since you have to pass `this` in for the context (or I might be wrong, you could try it).